lib/Dialect/FIRRTL/FIRRTLTypes.cpp

```diff
@@ -1126,6 +1180,230 @@ std::pair<uint64_t, bool> BundleType::rootChildFieldID(uint64_t fieldID,
                         fieldID >= childRoot && fieldID <= rangeEnd);
 }
 
+bool BundleType::isConst() { return getImpl()->isConst; }
+
+//===----------------------------------------------------------------------===//
+// OpenBundle Type
+//===----------------------------------------------------------------------===//
+
+struct circt::firrtl::detail::OpenBundleTypeStorage
+    : detail::FIRRTLBaseTypeStorage {
```

Comment:
May be clearer to just track isConst explicitly here instead of "inheriting" from the type storage from FIRRTLBaseType.